libsolutil/CMakeLists.txt

```diff
@@ -42,7 +42,7 @@ set(sources
 )
 
 add_library(solutil ${sources})
-target_link_libraries(solutil PUBLIC jsoncpp Boost::boost Boost::filesystem Boost::system range-v3)
+target_link_libraries(solutil PUBLIC Boost::boost Boost::filesystem Boost::system range-v3)
```

Comment:
So the new lib is header only. And it's a massive header. How does this affect compilation times?

Reply:
Good question.

Reply:
FYI, there's a forward declaration header (`json_fwd.hpp`) that can help with compile times. Since v3.11.x, there's also the added benefit of getting linker errors instead of incorrect behavior or crashes at runtime when inappropriately mixing different library versions or configurations.